### PR TITLE
Addition of button effect

### LINKED ISSUE #960: ✨[Feature Request]: button effect
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature

This feature will be added to all the webpages.

### Add ScreenShots

https://github.com/user-attachments/assets/6a78c870-a6ec-4530-8434-8aafee9eda9a

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Anjaliavv51**
#960

## Description
Effecr is added to all the buttons in the repo.

[Please include a brief description of the changes or features added]

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #960

## Type of PR

- [ ] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)

https://github.com/user-attachments/assets/6a78c870-a6ec-4530-8434-8aafee9eda9a

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by varshith257** [comment]
Why there is large gap between Rate us and appointment form and footer?

**Comment by Anjaliavv51** [comment]
@varshith257  their is no gap

**Comment by Anjaliavv51** [comment]
> Why there is large gap between Rate us and appointment form and footer?

@varshith257 their is no such gap , below is the preview of this PR.

https://github.com/user-attachments/assets/0b770833-7016-4e33-b7a0-d6aa615267f4

File: index.html [hunk 1]
```diff
@@ -126,11 +129,41 @@ <h1 style="font-size: 65px;">
...
       <h4>
-        <a href="#about" class="btn-get-started scrollto">Download Now</a>
+        <a href="#about" class="btn-get-started scrollto" id="btn">Download Now</a>
       </h4>
     </div>
-    <img class="sec-img" src="images/rd.png" alt="" />
+    <img class="sec-img" src="images/rd.png" alt="" id="sec-img" style="padding: 0 0 0 0;"/>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.11.4/gsap.min.js"></script>
+<script>
+  const img = document.getElementById('sec-img');
```
Comment (varshith257) [inline index.html:138]: Same here too..we can make code reusable

Comment (Anjaliavv51) [reply index.html:138]: sorry i didn't get u

Comment (Anjaliavv51) [reply index.html:138]: @varshith257 that is added by mistakely , that code should be added in tilt effect PR.

I removed it now

**Comment by Anjaliavv51** [comment]
@varshith257 now please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

**Review by varshith257 — APPROVED** [review]

**Comment by varshith257** [comment]
Can fix merge conflicts?

**Comment by Anjaliavv51** [comment]
> Can fix merge conflicts?

okay

**Comment by Anjaliavv51** [comment]
@varshith257 conflicts are resolved.

**Review by varshith257 — APPROVED** [review]

**Pull request merged**